test_extraction_DK.py

```diff
@@ -4,12 +4,88 @@
 import time
 from extract import extract_to_list
 from torch.utils.data import DataLoader
+from typing import Optional
 
-output_path = "results/epoch1_dev/step8000/carb_dev/"
-dev_gold = "evaluate/OIE2016_dev.txt"
+class RelationExtraction(): 
 
-model_path = "results/model-epoch1-step16000-score1.9768.bin"
+    def __init__(self, model_path: Optional[str] = None):
+        self._bert_config  = "bert-base-multilingual-cased"
+        self._binary = False
+        self._device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+        self._batch_size = 64
+        self._max_len=64
+        self._num_workers=4
+        self._args = {"bert_config": self._bert_config, "device": self._device, "binary": self._binary}
+        self._bert_model = self.prepare_model(model_path)
+        self._pin_memory = True
+    
 
+# Function for loading a model
+
+    def load_model(self, path):
```

Comment:
When a function is not supposed to be used by end users you usually prefix it with `_`
```suggestion
    def _load_model(self, path):
```